src/OpenLoco/src/Ui/ViewportInteraction.cpp

```diff
@@ -921,6 +921,10 @@ namespace OpenLoco::Ui::ViewportInteraction
                     {
                         interactionsToExclude |= InteractionItemFlags::building | InteractionItemFlags::headquarterBuilding | InteractionItemFlags::industry;
                     }
+                    if (vp->hasFlags(ViewportFlags::seeThroughBridges))
+                    {
+                        // TODO put bridges that are transluscent here
```

Comment:
is this a "todo for this pr" or a "todo in the future"

Reply:
Pretty sure it should mean this pr all you need to do is or the bridge into this

Reply:
Replaced with "interactionsToExclude |= InteractionItemFlags::bridge"

Reply:
Instead changed to "interactionsToExclude |= InteractionItemFlags::bridge"

Reply:
Ah yes i forgot about the flags being different. It should be doable to add bridge to the flags. But there might be a reason why you can't.

Reply:
Should I make an attempt to add InteractionItemFlags::bridge in this PR or is this just an issue to add?

Reply:
sorry i remember now what this code does. You don't need to do anything for bridge here as it can't be interacted. See my other comment.